tests/src/integration/masp.rs

```diff
@@ -0,0 +1,1205 @@
+use std::path::PathBuf;
+
+use color_eyre::eyre::Result;
+use color_eyre::owo_colors::OwoColorize;
+use namada_apps::client::tx::CLIShieldedUtils;
+use namada_core::types::address::{btc, eth, masp_rewards};
+use namada_core::types::token;
+use namada_core::types::token::{DenominatedAmount, NATIVE_MAX_DECIMAL_PLACES};
```

Comment:
```suggestion
use namada_core::types::token::{DenominatedAmount, NATIVE_MAX_DECIMAL_PLACES};
use test_log::test;
```
to allow capturing tracing logs with `RUST_LOG`